### ISSUE TITLE
swarm support

### ISSUE THREAD

**Issue by npcode** [rancher/convoy#91]
Is it possible to use convoy on swarm cluster?

**Comment by zbyte64** [comment]
Yes, I have done this. You will need convoy running on every node.

**Comment by npcode** [comment]
@zbyte64 Greate! I'll try it.

**Comment by deviantony** [comment]
Just tried it, works pretty well !

**Comment by zbyte64** [comment]
One thing I haven't tried with swarm is rescheduling: https://docs.docker.com/swarm/scheduler/rescheduling/

For EBS, convoy will need to detach the EBS volume from the original node before attaching it to a new node.

**Comment by yasker** [comment]
@zbyte64 We're adding that detach EBS function now...

**Comment by chrisgahlert** [comment]
Just confirmed: Rescheduling is working with NFS mounts! \o/
Deleting volumes in the Swarm on the other hand is problematic...

**Comment by RotemEmergi** [comment]
Hi ,
I am using convoy on top of nfs , I have some problem with mount .
did my compose is ok ? 

> version: '3'
networks:
  test:
    external: true
volumes:
   webapps:
    driver: convoy
   etc:
    driver: convoy
   agents:
    driver: convoy
services:
  webapps:
    image: XXXXXXXX
    networks:
        - test
    ports:
        - "80:8080"
        - "443:8443"
    volumes:
        - webapps:/var/lib/jetty/webapps
        - etc:/var/lib/jetty/etc
        - agents:/var/lib/jetty/agents
    depends_on:
        - rs1
        - rs2
        - rs3
    deploy:
      replicas: 3
  rs1:
    image: mongo:3.2
    networks:
        - test
    command: mongod --replSet "rs0"
    hostname: rs1
    volumes:
        - /opt/test/running/mongodb/rs1:/data/db
  rs2:
    image: mongo:3.2
    networks:
        - test
    command: mongod --replSet "rs0"
    hostname: rs2
    volumes:
        - /opt/test/running/mongodb/rs2:/data/db
  rs3:
    image: mongo:3.2
    networks:
        - test
    hostname: rs3
    command: mongod --replSet "rs0"
    volumes:
        - /opt/test/running/mongodb/rs3:/data/db
  rs:
    image: lucj/mongors
    networks:
        - test
